src/nodes.coffee

```diff
@@ -562,8 +628,8 @@ exports.Value = class Value extends Base
     lastProp instanceof Slice
 
   looksStatic: (className) ->
-    @base.value is className and @properties.length is 1 and
-      @properties[0].name?.value isnt 'prototype'
+    (@this or @base instanceof ThisLiteral or @base.value is className) and
+      @properties.length is 1 and @properties[0].name?.value isnt 'prototype'
```

Comment:
Hrm, I was assuming we would continue to use `this` assignment to define static methods. A `static` keyword doesn't really add anything.

So, this is checking if the variable base includes `this` (e.g. `@blah = ->`), or if the variable base is the class name (e.g. `Foo.blah = ->`), *and* that the second component isn't `prototype` (e.g. `Foo.prototype = ->`).